src/content/basics/nodepools/index.md

```diff
@@ -36,7 +41,7 @@ administrator.
 
 Node pools can be created when creating a cluster
 
-- via the Giant Swarm web interface ("happa")
```

Comment:
We actually expose that name "happa" in pages like these: https://docs.giantswarm.io/reference/web-interface/

But not really necessary to mention it here. Your choice.

Reply:
I mean to remove Happa in general after discussions about product naming.